snippets/settings.py

```diff
@@ -363,3 +363,5 @@
         },
     },
 }
+
+DATA_UPLOAD_MAX_NUMBER_FIELDS = config('DATA_UPLOAD_MAX_NUMBER_FIELDS', default=1000, cast=int)
```

Comment:
I don't see this setting referenced anywhere else in this diff.

Reply:
I have a comment in the commit about it but I also added one here. tl;dr when migrating the hundreds of snippets django raises a SuspiciousOperationError. We need to increase this number to make this work